apps/gallery/test/marionette/info_view_test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var Gallery = require('./lib/gallery.js'),
+    Fullscreen_View = require('./lib/fullscreen_view.js'),
+    GalleryInfoView = require('./lib/gallery_info_view.js'),
+    assert = require('assert');
+
+marionette('the gallery', function() {
+
+  var app, client, fullscreen_view, info_view;
+
+  client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    },
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    // Add file into the pictures directory
+    client.fileManager.add({
+      type: 'pictures',
+      filePath: 'test_media/Pictures/firefoxOS.png'
+    });
+    
+    app = new Gallery(client);
+    fullscreen_view = new Fullscreen_View(client);
+    info_view = new GalleryInfoView(client);
+    app.launch();
+  });
+
+  test('check displayed information of image', function() {
+    assert.strictEqual(app.thumbnails.length, 1);
+    app.tapFirstThumbnail();
+    assert.ok(fullscreen_view.displayed);
```

Comment:
oh actually, apparently displayed property does the wait call inside:
https://github.com/npark-mozilla/gaia/blob/1242615/apps/gallery/test/marionette/lib/fullscreen_view.js#L89